web/src/OverviewPane.tsx

```diff
@@ -40,31 +40,44 @@ let ServicesContainer = styled.div`
   overflow: auto;
 `
 
-function PinnedServicesDivider() {
-  return (
-    <ServicesDividerRoot>
-      <GridDividerPinSvg style={{ marginLeft: "28px" }} />
-      <ServicesLabel>Pinned Resources</ServicesLabel>
-    </ServicesDividerRoot>
-  )
+type ResourceProps = {
+  items: OverviewItem[]
 }
 
-function AllResourcesDivider() {
-  return (
-    <ServicesDividerRoot>
-      <GridDividerAllSvg style={{ marginLeft: "28px" }} />
-      <ServicesLabel>All Resources</ServicesLabel>
-    </ServicesDividerRoot>
-  )
+export function PinnedResources(props: ResourceProps) {
+  return props.items?.length ? (
+    <React.Fragment>
+      <ServicesDividerRoot className={"ServicesDivider pinned"}>
```

Comment:
you should be able to get rid of these classes too now?

Reply:
whoops good catch